apps/webcert/src/store/certificate/certificateThunks.ts

```diff
@@ -1,9 +1,11 @@
 import { type Certificate } from '../../types'
 import { getCertificateToSave } from '../../utils'
 import { createAsyncApiThunk } from '../api/createAsyncApiThunk'
+import { toggleCertificateFunctionDisabler } from './certificateActions'
 
 export const autoSaveCertificate = createAsyncApiThunk<{ version: number }, Certificate>('certificate/autosave', (certificate) => ({
   url: `/api/certificate/${certificate.metadata.id}`,
   method: 'PUT',
   data: getCertificateToSave(certificate),
+  functionDisablerType: toggleCertificateFunctionDisabler.type,
```

Comment:
Adds autosave request to the "toggleCertificateFunctionDisabler" function disable group. Sign button should be disabled when autosave is pending.